gossip/client_test.go

```diff
@@ -95,7 +93,7 @@ func TestClientGossip(t *testing.T) {
 	client := newClient(remote.is.NodeAddr)
 	// Use an insecure context. We're talking to unix socket which are not in the certs.
```

Comment:
nevermind, this is a comment that somehow got out of date. Luckily, my PR (https://github.com/cockroachdb/cockroach/pull/1408/files) re-introduces insecureTestBaseContext right below it.